### PR TITLE
[RAC-6136] update config file due to RackHD install in ubuntu 16.04

### PR DESCRIPTION
**Pull request by HQebupt**
Update config file due to RackHD install from source code in ubuntu 16.04.
- Related PR: [/docs/pull/405](https://github.com/RackHD/docs/pull/405/files)
@changev @PengTian0 @panpan0000

### CODE REVIEW

File: packer/ansible/roles/images/tasks/main.yml [hunk 1]
```diff
@@ -13,9 +13,14 @@
            validate_certs=no
            force=yes
   with_items:
-   - discovery.overlay.cpio.gz
    - base.trusty.3.16.0-25-generic.squashfs.img
+   - discovery.docker.tar.xz
```
Comment (panpan0000) [inline packer/ansible/roles/images/tasks/main.yml]: Good job !

File: packer/ansible/roles/mongodb/tasks/main.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 - name: Install MongoDB
   apt: pkg={{ item }} state=installed
   with_items:
-    - mongodb=1:2.4.9-1ubuntu2
+    - mongodb=1:2.6.10-0ubuntu1
```
Comment (panpan0000) [inline packer/ansible/roles/mongodb/tasks/main.yml]: what's the reason to switch to 2.6.10?

Comment (HQebupt) [reply packer/ansible/roles/mongodb/tasks/main.yml]: Confirmed with @changev , we can use the default mongodb version in both 14.04 & 16.04 ubuntu.

**Comment by changev** [comment]
👍

**Pull request merged**